modules/oc-mirror-support.adoc

```diff
@@ -10,3 +10,9 @@
 The oc-mirror plugin supports mirroring {product-title} payload images and Operator catalogs for {product-title} versions 4.9 and later.
 
 Use the latest available version of the oc-mirror plugin regardless of which versions of {product-title} you need to mirror.
+
+// TODO: Remove this in 4.14
+[IMPORTANT]
+====
+If you used the Technology Preview OCI feature for the oc-mirror plugin for OpenShift Container Platform 4.12, you can no longer use the OCI feature of the oc-mirror plugin to create an image set archive as a first step to mirroring to a fully disconnected cluster.
```

Comment:
In 4.12, it wasnt creating a tar... it was just copying the catalog in oci format to the disk (with a bit of preparing the content - like filtering operators, etc): but all in all something closer to what `skopeo copy` would do, rather than a proper image set archive
```suggestion
If you used the Technology Preview OCI feature for the oc-mirror plugin for OpenShift Container Platform 4.12, you can no longer use the OCI feature of the oc-mirror plugin to copy a catalog locally and convert it to oci format as a first step to mirroring to a fully disconnected cluster.
```

Reply:
Thanks, will update the few other places we have this statement as well